tensorflow/lite/micro/tools/ci_build/test_project_generation.sh

```diff
@@ -87,3 +87,33 @@ PATH=${PATH}:${ROOT_DIR}/tensorflow/lite/micro/tools/make/downloads/gcc_embedded
 popd > /dev/null
 
 #rm -rf ${TEST_OUTPUT_DIR_CMSIS}
+
+# Test that C++ files are renamed to .cpp
+rm -rf ${TEST_OUTPUT_DIR_CMSIS}
+
+# Remove existing state prior to testing project generation for cortex-m target.
+# make -f tensorflow/lite/micro/tools/make/Makefile clean clean_downloads
+
+TEST_OUTPUT_DIR_CMSIS=$(mktemp -d)
+
+readable_run \
+  python3 tensorflow/lite/micro/tools/project_generation/create_tflm_tree.py \
+  --makefile_options="TARGET=cortex_m_generic OPTIMIZED_KERNEL_DIR=cmsis_nn TARGET_ARCH=project_generation" \
```

Comment:
my preference would be to have this not use any additional makefile_options, i.e. use

```
readable_run \
  python3 tensorflow/lite/micro/tools/project_generation/create_tflm_tree.py \
  ${TEST_OUTPUT_DIR_RENAME_CC}
```